content/influxdb/cloud-iox/reference/sql/functions/string.md

```diff
@@ -0,0 +1,1114 @@
+---
```

Comment:
This looks like great content. Would you be amenable to figuring out how we could move the content into the DataFusion repo (and maybe have a script to sync it over here or something)? That way we could get more reviewers in the community to help review

Reply:
My plan is to port all of this content over to the DataFusion docs once it's complete. That will likely be sometime next week. Pulling content from upstream will be a bit of a challenge since not all functionality upstream is applicable downstream in InfluxDB, but we may be able to figure something out.

Reply:
Filed https://github.com/influxdata/idpe/issues/17251 to track helping this effort